src/core/pages/PatientSummary/functions.js

```diff
@@ -0,0 +1,43 @@
+import get from "lodash/get";
```

Comment:
'function' is a very bad file name, you should find a better one, like e.g. 'getSummaryContainerStyles.js'